src/components/DateField.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+import 'date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import PropTypes from 'prop-types';
+
+function DateField(props) {
+  const { id, label, value, handler } = props;
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <KeyboardDatePicker
+        id={id}
```

Comment:
Use `{...props}` to pass in fields of the same name, i.e. id, label, value, onChange